stdlib/public/Bionic/Glibc.swift

```diff
@@ -0,0 +1,215 @@
+//===----------------------------------------------------------------------===//
```

Comment:
OK, managed to figure this one out: https://github.com/apple/swift/pull/1679 -- I'd definitely appreciate feedback on that approach! :bow: